assets/views/home/header.ejs

```diff
@@ -33,6 +33,7 @@
             <li><a href="#" data-toggle="modal" data-target="#flairAppList">Flair App List</a></li>
             <li><a href="#" data-toggle="modal" data-target="#flairMod">Edit Flairs</a></li>
             <li><a href="/mod/banuser">Ban a User</a></li>
+            <li><a target="_self" href="/mod/downloadModmail" download="modmails.txt">Download Modmails</a></li>
```

Comment:
¯\\\_(ツ)_/¯

I guess it might be useful if people want to do some very specific type of search/data collection/statistical analysis that we don't provide. I think we should offer the entire file for download somewhere, although I'd be fine with putting it in a less prominent location.